pkg/config/protocol/instance.go

```diff
@@ -85,6 +87,8 @@ func Parse(s string) Instance {
 		return Redis
 	case "mysql":
 		return MySQL
+	case "postgres":
```

Comment:
need to make it opt-in with an env var, like mysql and others

Reply:
in mysql and others - we are actually adding the specific filters like mysql. Here we are not adding them. Do you still want to control whether startTLS is added based on that?

Reply:
I think it may be backwards incompatible otherwise? Plus, I would guess in the future we would add the filters(?)

Reply:
Sure. that makes sense. I can add that flag